### PR TITLE
#93 Fix disable_dialog_boxes

### PR DESCRIPTION
**Pull request by kettenbach-it**

### CODE REVIEW

**Comment by kettenbach-it** [comment]
@hkpeprah Please approve!

File: pylink/jlink.py [hunk 1]
```diff
@@ -968,14 +965,12 @@ def enable_dialog_boxes(self):
           ``None``
         """
         self.exec_command('SetBatchMode = 0')
+        self.exec_command("HideDeviceSelection = 0")
```
Comment (hkpeprah) [inline pylink/jlink.py:968]: I think you'll also need to re-enable the info windows: `EnableInfoWinFlashBPs`, and `EnableInfoWinFlashDL`.

**Review by hkpeprah — CHANGES_REQUESTED** [review]
Sorry about the late review.  Only thing that needs changing is to re-enable the flash dialog popups when `enable_dialog_boxes()` is called.  Otherwise LGTM.

**Comment by kettenbach-it** [comment]
@hkpeprah Please approve!

**Review by hkpeprah — APPROVED** [review]
Sorry for the delay here.  Just tested out locally, and LGTM.  Will merge and cut a minor release later today.

**Comment by hkpeprah** [comment]
Fix should be present in `v0.10.0`.

**Pull request merged**